README.md

```diff
@@ -52,8 +52,8 @@ ensure org-mode buffers are not dimmed.
 * `dimmer-configure-posframe` is a convenience function for posframe
 users to ensure posframe buffers are not dimmed.
 
-* `dimmer-configure-which-key` is a convenience function for which-key
-users to ensure which-key popups are not dimmed.
```

Comment:
It seems you accidentally deleted the documentation for dimmer-configure-which-key. 

And actually, I'm trying to keep these dimmer-configure-xxx functions in alphabetical order, so can this move up to the appropriate place in the list?  Same for the header comment in the source file.